app/grandchallenge/reader_studies/views.py

```diff
@@ -847,12 +846,13 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class AnswerBatchDelete(LoginRequiredMixin, DeleteView):
+class AnswerBatchDelete(LoginRequiredMixin, SuccessMessageMixin, FormView):
```

Comment:
I think you're missing a `template_name` for this view to work.

Reply:
Also, the `FormView` does not except `DELETE` as a method, so you need to change the method to `POST` in `remove_answers.js`

Reply:
Also needs to be changed in `readerstudy_detail.html` for deleting the ground truth

Reply:
> I think you're missing a template_name for this view to work.

OK that's strange as the test for it pass.

Reply:
I misdiagnosed the problem originally. The issue is with the method, not the missing template name. Sorry, forgot to delete the first comment.

Reply:
But still strange that the tests pass, or do they use `client.post`?

Reply:
They were updated to use `client.post`.